package.json

```diff
@@ -74,7 +74,7 @@
     "@babel/preset-react": "^7.22.5",
     "@babel/preset-typescript": "^7.22.11",
     "@babel/types": "^7.22.11",
-    "@commitlint/cli": "^17.7.1",
+    "@commitlint/cli": "^17.7.2",
```

Comment:
Is this change required for commitlint/cli?

Reply:
Yes, this change is required for commitlint/cli. I made these adjustments to ensure that commit messages adhere to our defined format and meet the commitlint criteria, enhancing the consistency and quality of our commit messages
This change is required because of this error:-

`At line:1 char:56
+ npm install --save-dev @commitlint/{config-conventional,cli}
+                                                        ~
Missing argument in parameter list.
    + CategoryInfo          : ParserError: (:) [], ParentContainsErrorRecordException
    + FullyQualifiedErrorId : MissingArgument`